packages/odyssey-react-mui/src/Tabs.tsx

```diff
@@ -53,16 +54,21 @@ export type TabsProps = {
    */
   ariaLabel?: string;
   /**
-   * The value of the Tab that should be selected by default
+   * The value of the Tab that should be selected by default.
+   * If value is provided as well, value takes precedence.
    */
   initialValue?: string;
   /**
    * The TabItems to be included in the Tabs group
    */
   tabs: TabItemProps[];
+  /**
+   * The value of the Tab that is selected
+   */
+  value?: string;
 };
 
-const Tabs = ({ ariaLabel, initialValue = "0", tabs }: TabsProps) => {
+const Tabs = ({ ariaLabel, initialValue = "0", tabs, value }: TabsProps) => {
   const [tabState, setTabState] = useState(initialValue);
```

Comment:
Do we want to fallback on `value` if `initialValue` isn't passed? It's a super simple change that prevents and extra initial rerender?
```suggestion
const Tabs = ({ ariaLabel, initialValue = "0", tabs, value }: TabsProps) => {
  const [tabState, setTabState] = useState(initialValue ?? value);
```